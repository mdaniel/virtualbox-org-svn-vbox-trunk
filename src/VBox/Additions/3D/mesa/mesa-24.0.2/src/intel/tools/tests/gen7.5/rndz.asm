rndz(8)         g8<1>.xF        g1<0>.xF                        { align16 1Q };
rndz(8)         g54<1>F         g43<8,8,1>F                     { align1 1Q };
rndz(16)        g98<1>F         g95<8,8,1>F                     { align1 1H };
