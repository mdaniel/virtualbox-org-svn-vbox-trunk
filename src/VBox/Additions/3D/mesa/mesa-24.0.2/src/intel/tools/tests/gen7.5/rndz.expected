47 01 60 00 bd 03 01 21 20 00 00 00 00 00 00 00
47 00 60 00 bd 03 c0 26 60 05 8d 00 00 00 00 00
47 00 80 00 bd 03 40 2c e0 0b 8d 00 00 00 00 00
